src/ui/game/Game.tsx

```diff
@@ -242,44 +252,32 @@ export default () => {
         }
       )}
       style={{ ['--aspect-ratio' as string]: game.board._ui.boardSize.aspectRatio }}
-      onClick={() => finished && setVictoryMessageDismissed(true)}
     >
       <audio ref={clickAudio} src={click} id="click"/>
-      <div id="background"/>
+      <div id="background" className="full-page-cover" />
       <div id="play-area" style={{width: '100%', height: '100%'}} className={dragElement ? "in-drag-movement" : ""}>
         {mode !== 'debug' && (
           <Element
             element={game.board}
             json={boardJSON[0]}
-            mode={mode}
+            mode={announcement ? 'info' : mode}
             onSelectElement={onSelectElement}
             onSelectPlacement={onSelectPlacement}
           />
         )}
       </div>
 
-      {mode === 'game' && <PlayerControls
-        name={name}
-        style={style}
-        moves={moves}
-        onSubmit={submitMove}
-      />}
-
-      {game.godMode && mode === 'game' && <div className="god-mode-enabled">God mode enabled</div>}
-
-      {finished && !victoryMessageDismissed && (
-        <div className="game-finished">
-          Game finished
-          {game.winner.length > 0 && (
-            <div style={{color: game.winner.length === 1 ? game.winner[0].color : ''}}>
-              {game.winner.map(p => p.name).join(', ')} win{game.winner.length === 1 && 's'}!
-            </div>
-          )}
-          {game.winner.length === 0 && game.players.length > 1 && <div>Tie game</div>}
-          {game.winner.length === 0 && game.players.length === 1 && <div style={{color: "#800"}}>You lose</div>}
-        </div>
+      {mode === 'game' && !announcement && (moves.length || !game.players.currentPosition.includes(position)) && (
```

Comment:
correctly hide prompts if this is the current player but no moves are available currently (animation delays or autoplay)